doc/network.tex

```diff
@@ -135,107 +138,105 @@
 This concept of layers is misleading when discussing Ouroboros.
-It is {\em not} the case that the consensus (layer) is build on top of the network (layer).
+For example, it is {\em not} the case that the consensus (layer) is built on top of the network (layer).
 It is more appropriate to talk about a network component than a network layer.
-The network component provides services to the consensus component and vice versa.
-Both components rely on each other.
-The network component uses the consensus component validate
-information that it distributes to other nodes.
-This is essential to guard against certain kinds of DoS attacks.
+The network component provides services to the consensus component and vice versa;
+both components rely on each other.
+The network component uses the consensus component to validate
+information that it distributes to other nodes, which
+is essential to guard against certain kinds of DoS attacks.
 Existing peer-to-peer systems focus on a slightly different problem domain.
 For example, they do not consider the information validation issue
-or have worries \todo{(Neil : I could not read the word)}, such as 'eclipse' that to not
+or are concerned with issues such as 'eclipse attacks' that to not
 apply to the Ouroboros family of protocols.
 
 \missingfigure[figwidth=6cm]{Testing a long text string}
 
 \subparagraph{Performance of Peer to Peer Networks}
-In computer science, Byzantine Fault Tolerance is a property which states
-that a distributed algorithm works for the honest participants of the algorithm
-under the assumption that a certain percentage of the participants is indeed honest.
+In computer science, Byzantine Fault Tolerance is a property of a distributed algorithm, which states
+that it works for the honest participants
+under the assumption that a certain proportion of the participants are indeed honest.
 A similar, but more informal property applies to peer to peer networks as well.
 
-A peer to peer network, is a system which provides a service to its participants while at the same
-time the participants of the network provide a service to the network.
+A peer to peer network is a system that provides a service to its participants while at the same
+time the participants provide a service to the network.
 The performance of the P2P network depends (among other things) on the performance of the nodes,
 while the performance of a node also depends on the performance of the network.
 Not only are there honest and adversarial participant, but there is also a huge variety of
 possible network topographies, bandwidths and latencies of network connections and other
 factors that determine the performance of the network.
-This document discusses the high level business requirements for Cardano and what are the
-assumptions on the structure of the underlying P2P network.
+
+This document discusses the high level business requirements for Cardano and the
```

Comment:
I think 'buisness' is a bit overblowing it! "functional and performance" might be a better fit